src/core/js/fixes/img.lazyload.js

```diff
@@ -0,0 +1,40 @@
+define([
+  'core/js/adapt',
+  '../templates'
+], function(Adapt) {
+
+  /**
+   * 27 April 2020 https://github.com/adaptlearning/adapt_framework/issues/2734
+   * Chrome on Android defers load events on images when lite mode is enabled
+   * and as part of a data saving technique.
+   *
+   * Add a loading="eager" attribute to all template and partial img tags where
+   * the loading attribute is missing.
+   */
+  Adapt.on('app:dataReady', function() {
+    const config = Adapt.config.get('_fixes');
+    if (config && config._imgLazyLoad === false) return;
+    applyImgLoadingFix();
+  });
+
+  function applyImgLoadingFix() {
+    const findImgTag = /<img([^>]*)>/gi;
+    const hasLoadingAttr = / loading=/gi;
+    Adapt.on('template:postRender partial:postRender', function(event) {
+      const imgs = event.value.match(findImgTag);
+      if (!imgs) {
+        // No img tags found
```

Comment:
comment seems a bit unnecessary... if you feel it needs clarity maybe rename variable to `imgTagsFound`? but I think it's clear enough already